lib/Doctrine/Common/Cache/PhpFileCache.php

```diff
@@ -106,4 +98,23 @@ protected function doSave($id, $data, $lifeTime = 0)
 
         return $this->writeFile($filename, $code);
     }
+
+    /**
+     * @param string $id
+     *
+     * @return array|false
+     */
+    private function includeFileForId($id)
+    {
+        $fileName = $this->getFilename($id);
+
+        // note: error suppression is still faster than `file_exists`, `is_file` and `is_readable`
+        $value = @include $fileName;
```

Comment:
Yea, I guess so. Too bad. Its not like this will actually be any better in PHP 7 with exceptions either (for actual errors, not warnings) since exceptions aren't cheap either.